sklearn/preprocessing/_polynomial.py

```diff
@@ -936,7 +936,6 @@ def transform(self, X):
             spl = self.bsplines_[i]
 
             if self.extrapolation in ("continue", "error", "periodic"):
-
```

Comment:
There is a left-over line deletion in `sklearn/preprocessing/_polynomial.py` -- could you undo this?